easyprivacy/easyprivacy_thirdparty.txt

```diff
@@ -1235,6 +1235,7 @@
 ||metrics.ctvdigital.net^
 ||metrics.kmsmep.com^
 ||metrics.mdstrm.com^
+||metrics.onshape.com^
```

Comment:
You should add `tracking.go.onshape.com`  as well (https://mypdns.org/my-privacy-dns/matrix/-/issues/4122652#hosts-and-pi-hole)